common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2366,8 +2366,11 @@ public static enum ConfVars {
         "Whether Hive enables the optimization about converting common join into mapjoin based on the input file size. \n" +
         "If this parameter is on, and the sum of size for n-1 of the tables/partitions for a n-way join is smaller than the\n" +
         "specified size, the join is directly converted to a mapjoin (there is no conditional task)."),
-    HIVE_CONVERT_ANTI_JOIN("hive.auto.convert.anti.join", true,
-        "Whether Hive enables the optimization about converting join with null filter to anti join"),
+    HIVE_CONVERT_ANTI_JOIN("hive.auto.convert.anti.join", false,
+        "Whether Hive enables the optimization about converting join with null filter to anti join. The " +
+        "default is currently false because of HIVE-25669. There is the possibility of incorrect results. " +
```

Comment:
@scarlin-cloudera I suppose here you meant to point to HIVE-26659 (not HIVE-25669).